BuildRelease.msbuild

```diff
@@ -107,7 +107,7 @@
         <ItemGroup>
           <XunitTestAssemblies
               Include="Src\*Test\bin\$(Configuration)\*Test.dll"
-              Exclude="Src\AutoFixture.NUnit2.*Test\bin\$(Configuration)\*.dll" />
+              Exclude="Src\AutoFixture.NUnit*.*Test\bin\$(Configuration)\*.dll" />

```

Comment:
https://ci.appveyor.com/project/AutoFixture/autofixture/build/1.0.131-ftldqexm
(line 8668 - 8696)
The build log shows that all xUnit tests are run, as well as Ploeh.AutoFixture.NUnit2.UnitTest.dll, but not Ploeh.AutoFixture.NUnit3.UnitTest.dll which is not set up with a NUnit3 build task and should be excluded at this moment.

Does that makes sense?